cmd/marker/Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM centos:centos7
-RUN yum install -y openvswitch && yum clean all
+FROM fedora:29
```

Comment:
could you please change the title of the commit to say that we are using newer ovs? the current title is misleading

Reply:
done